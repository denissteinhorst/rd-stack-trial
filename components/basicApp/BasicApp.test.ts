// @vitest-environment nuxt
import { shallowMount } from '@vue/test-utils';
import { describe, expect, test } from 'vitest';
import Component from './BasicApp.vue';

describe('BasicApp (Unit)', () => {
    test('renders correctly', () => {
        const wrapper = shallowMount(Component);
        expect(wrapper.html()).toMatchSnapshot();
    });
});