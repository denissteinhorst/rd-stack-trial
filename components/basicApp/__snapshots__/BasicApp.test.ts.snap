// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`BasicApp (Unit) > renders correctly 1`] = `""`;
